sources/packages/web/src/services/http/Institution.ts

```diff
@@ -115,4 +115,23 @@ export class InstitutionApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async checkIfExist(guid: string, headers: any): Promise<boolean> {
+    try {
+      const resp: AxiosResponse<void> = await this.apiClient.head(
+        "institution/" + guid,
+        headers,
+      );
+      if (200 === resp.status) {
```

Comment:
Don't need to check for 200 at this point. If the request was not successful Axios will throw an exception, so at this point, it is safe to return true.

Reply:
Fixed.